web-wallet/src/lib/transactions/transactions.d.ts

```diff
@@ -4,5 +4,6 @@ type Transaction = {
   direction: "In" | "Out";
   fee: number;
   id: string;
+  memo?: string | Uint8Array;
```

Comment:
Two questions: 
- are you sure that this can be an Uint8Array?
- are you sure that the field is optional? Not sure, but I thought it's always there but can be `null`.